regexes/device/mobiles.yml

```diff
@@ -7820,7 +7847,7 @@ K-Touch:
 
 # kyocera (kyocera.co.jp, kyocera.com)
 Kyocera:
-  regex: 'Kyocera|KWC-|QC-|(?:NP501KC|302KC|503KC|602KC|705KC|901KC|C51[27]0|C5155|C5215|C67[35]0|C6522N?|C6530N|C6740N?|C6743|E6[89]10|KYL2[13]|KYV3[13-9]|KYT3[123]|KYF3[123579]|KYX31|KYY2[1-4]|S2151|KC-S701|S4-KC|KYV40U?|KYV4[1-8]|KYV4[47]|KC-S301AE|X3-KC|S6-KC|E6560[CT]|E6782[L]?|E6790(?:TM)?|[47]04KC|A001KC)(?:[-_]u2?)?(?:[);/ ]|$)'
```

Comment:
Wouldn't that remove detections for useragents like
`QC-2235/1.0.52 UP.Browser/4.1.22b1` (event though the detection might currently not be correct, as it removes the `QC`)

Reply:
Just looked up a useragent containing QC 🙈 Might be good if we would still detect them if possible

Reply:
@sgiehl 
added tests for devices `QC-[0-9]{4}`, which I managed to find from the list
https://deviceatlas.com/device-data/devices/kyocera